### PR TITLE
chore: update scaffolding dependencies and upgrade node from v16 to v20

### PR DESCRIPTION
**Pull request by spykr**
Next.js/Storybook were updated to prevent issues with the upgrade to Node v20.

### CODE REVIEW

**Comment by RickMeasham** [comment]
I am looking forward to this update. I have attempted it locally a couple of times

File: template/next-ts-chakra-ui/.storybook/main.ts [hunk 1]
```diff
@@ -1,29 +1,27 @@
...
+import type { StorybookConfig } from '@storybook/nextjs';
 
-module.exports = {
+const config: StorybookConfig = {
+  framework: '@storybook/nextjs',
   staticDirs: ['../public'],
   stories: [
-    '../node_modules/@portablestudios/figma2theme/lib/**/*.stories.js',
+    // TODO: Update figma2theme and uncomment this
+    // '../node_modules/@portablestudios/figma2theme/lib/**/*.stories.js',
```
Comment (RickMeasham) [inline template/next-ts-chakra-ui/.storybook/main.ts:9]: Once this is back, is there a way to make a private repository not be a dependency of this open source project?

Comment (spykr) [reply template/next-ts-chakra-ui/.storybook/main.ts:9]: Nice to see you @RickMeasham 😄 `figma2theme` is already a public repository! Though I wasn't aware anyone outside of Portable was using it (or this). Either way it's an optional dependency on the generated project so easy to remove if not needed.

https://github.com/PortableStudios/figma2theme

**Comment by RickMeasham** [comment]
Oops! I swear it used to complain. Cheers mate.On 13 Jun 2024, at 18:20, Darcy Glennen ***@***.***> wrote:﻿
@spykr commented on this pull request.



In template/next-ts-chakra-ui/.storybook/main.ts:
>    staticDirs: ['../public'],
   stories: [
-    ***@***.***/figma2theme/lib/**/*.stories.js',
+    // TODO: Update figma2theme and uncomment this
+    // ***@***.***/figma2theme/lib/**/*.stories.js',

Nice to see you @RickMeasham 😄 figma2theme is already a public repository! Though I wasn't aware anyone outside of Portable was using it (or this). Either way it's an optional dependency on the generated project so easy to remove if not needed.
https://github.com/PortableStudios/figma2theme

—Reply to this email directly, view it on GitHub, or unsubscribe.You are receiving this because you were mentioned.Message ID: ***@***.***>
Message  protected for iSite by MailGuard: e-mail anti-virus, anti-spam and content filtering.https://www.mailguard.com.au
Report this message as spam

  Message  protected for iSite by MailGuard: e-mail anti-virus, anti-spam and content filtering.https://www.mailguard.com.au

**Comment by spykr** [comment]
@RickMeasham It complains if you haven't authenticated with GitHub because that's necessary even if the package is public, we'd have to move the package to NPM to get around that 🫤

**Review by RuthAdele — APPROVED** [review]

**Comment by spykr** [comment]
> @RickMeasham It complains if you haven't authenticated with GitHub because that's necessary even if the package is public, we'd have to move the package to NPM to get around that 🫤

Done! https://github.com/PortableStudios/easey/pull/430

**Pull request merged**